src/Middlewares/PermissionMiddleware.php

```diff
@@ -24,12 +26,27 @@ public function handle($request, Closure $next, $permission, $guard = null)
 
         $permissions = is_array($permission)
             ? $permission
-            : explode('|', $permission);
+            : explode(self::SEPARATOR, $permission);
 
         if (! $user->canAny($permissions)) {
             throw UnauthorizedException::forPermissions($permissions);
         }
 
         return $next($request);
     }
+
+    /**
+     * Specify the permission and guard for the middleware.
+     *
+     * @param  array|string  $permission
+     * @param  string|null  $guard
+     * @return string
+     */
+    public static function using($permission, $guard = null)
+    {
+        $permissionString = is_string($permission) ? $permission : implode('|', $permission);
```

Comment:
in some places you use `self::SEPARATOR` and on others just `'|'`
I think that using `public const SEPARATOR = '|';` is unnecessary in something so simple, just use `'|'` and remove the const var

Reply:
Yeah, I figured it would be slightly neater to define this in a single place (e.g. pick one of the three and define it there), then figured calling the consts from another file would also be ugly, then I ended up in this inconsistent place. 

I agree defining the const is a bit overcomplicated for now, so I adjusted to stick with just using `'|'` everywhere.